assemblies/DataSources/OpenConfig/Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.xml

```diff
@@ -0,0 +1,996 @@
+            Gets the callback action to execute when an update is received.
+            </summary>
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Models.GenericSubscriptionDetails`2.Request">
+            <summary>
+            Gets the request to execute the subscription with.
+            </summary>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Models.GenericSubscriptionDetails`2.GetSubscribeSubject(Skyline.DataMiner.Protobuf.CommunicationGateway.Grpc.Api.Model.v1.ChannelId)">
+            <inheritdoc/>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Models.GenericSubscriptionDetails`2.GetSubscribeRequest">
+            <inheritdoc/>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Models.GenericSubscriptionDetails`2.Parse(Google.Protobuf.ByteString)">
+            <summary>
+            Parses the content from a ResponseMessage to the type matching the subscription as defined by the generic TResponse.
+            </summary>
+            <param name="data"></param>
+            <returns></returns>
+        </member>
+        <member name="M:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Models.GenericSubscriptionDetails`2.HandleUpdate(Google.Protobuf.ByteString)">
+            <inheritdoc/>
+        </member>
+        <member name="T:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Models.HeartbeatSettings">
+            <summary>
+            A wrapper for the heartbeat WatchdogTimer settings.
+            </summary>
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Models.HeartbeatSettings.DxmTimeoutMs">
+            <summary>
+            The timeout time in ms for a heartbeat to arrive from the DxM (CommunicationGateway).
+            </summary>
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Models.HeartbeatSettings.DxmTimeoutHysteresisMs">
+            <summary>
+            Specifies the minimum delay between timer resets to minimize overhead during high-activity periods from the DxM.
+            </summary>
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Models.HeartbeatSettings.HeartbeatDelayMs">
+            <summary>
+            The timeout time in ms for a heartbeat to be sent to the DxM (CommunicationGateway).
+            </summary>
+        </member>
+        <member name="P:Skyline.DataMiner.DataSources.CommunicationGatewayMiddleware.Common.Models.HeartbeatSettings.HeartbeatHysteresisMs">
+            <summary>
+            Specifies the minimum delay between timer resets to minimize overhead during high-activity periods to the DxM.
```

Comment:
How does "to the DxM" fit in this sentence? It isn't clear to me what this refers to. (same for line 772/773 below)
Is this what you mean?
```suggestion
            Specifies the minimum delay between timer resets to minimize overhead during high-activity periods of communication towards the DxM.
```